### ISSUE TITLE
vad_extract filter out speech

### ISSUE THREAD

**Issue by neil3212080** [hcmlab/vadnet#4]
hey sir ,when i use vad_extract with chinese one word,it can not discriminate speech and noise

**Comment by neil3212080** [comment]
-

**Comment by neil3212080** [comment]
/home/neil/code/vadnet-master/data/archive_name.tar

**Comment by neil3212080** [comment]
[archive_name.zip](https://github.com/hcmlab/vadnet/files/2459516/archive_name.zip)

**Comment by frankenjoe** [comment]
We tested `VadNet` with ifferent languages and actually got pretty good performance with Chinese. Yet, the model we ship was trained mainly on English/German. Probably, you will have to (re)train with Chinese samples to improve the results. Since the training code has been released (see `train` folder) we leave the job to you :)

**Comment by neil3212080** [comment]
if i want train chinese,just change train/download.py url?or Is there anything else that needs to be modified?

**Comment by frankenjoe** [comment]
Yes, just make sure you're media files come with subtitles. The following sites are supported: https://rg3.github.io/youtube-dl/supportedsites.html

**Comment by neil3212080** [comment]
Hello frankenjoe When I continue to use your model for testing, I found that in the case of a Chinese word, the false positive rate is 28%. I guess the reason for this is not a training set is not sensitive to a word. I want to try to repeat the two words and continue to do the test. Do you think this idea works? Or do you have any better suggestions to improve the accuracy of the model against a word?
